Dockerfile

```diff
@@ -94,21 +91,35 @@ ADD requirements.txt /requirements.txt
 ADD dev-requirements.txt /dev-requirements.txt
 # Install dependencies
 # TODO: Add --std=c++14
+ARG GPU_ARCH=";"
+ARG PREFIX=/usr/local
+ARG USE_FIN="OFF"
+ARG CCACHE_SECONDARY_STORAGE="redis://10.236.118.172"
```

Comment:
This should only be set in the jenkinsfile. We dont want users caching there builds here.

Reply:
This is temporary

Reply:
Fixed